src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const SITE_TITLE = "MapLibre";
+export const SITE_DESCRIPTION =
+  "The MapLibre Organization is an umbrella for open-source mapping libraries.";
```

Comment:
I saw this on `src/pages/index.astro` and didn't want to repeat it elsewhere since it's also needed on the RSS feed. I moved the string here. Feel free to move somewhere else or do it differently.